src/common/modals/AddInstance/InstanceName.js

```diff
@@ -103,6 +122,7 @@ const InstanceName = ({
 
     const isCurseForgeModpack = Boolean(version?.source === CURSEFORGE);
     const isFTBModpack = Boolean(modpack?.art);
+    const isModrinthModpack = Boolean(modpack?.project_type);
     let manifest;
 
     // If it's a curseforge modpack grab the manfiest and detect the loader
```

Comment:
Misspell in code comment

Reply:
I did not write that comment, it's not part of this PR.